Automattic/sensei
# Fix custom button colors not being applied in Learning Mode

Resolves #6995.

## Proposed Changes
- Remove the `!important` from the button color properties.
- Load custom CSS for Astra when Learning Mode is enabled to handle overrides.

_Note: This PR does not attempt to handle borders and border colors, nor try to fix any mismatch between the editor and the frontend when the default colors are used._

## Testing Instructions


- Activate Astra.
- Enable Learning Mode.
- Go to the lesson editor.
- Add 2 buttons - one with the default style and the other with outline style.
- View the lesson on the frontend and check that the buttons look as they did before this change.
- Set custom background and text colors for the buttons.
- View the lesson on the frontend and check that the buttons use the custom colors.
- Repeat the above with the Divi and Course themes or any other popular Sensei theme - P6rkRX-2wW-p2.

## Pre-Merge Checklist
